accel-pppd/radius/serv.c

```diff
@@ -197,11 +197,6 @@ int rad_server_req_enter(struct rad_req_t *req)
 
 	pthread_mutex_lock(&req->serv->lock);
 
-	if (ts.tv_sec < req->serv->fail_time) {
```

Comment:
pthread_mutext_lock() can take a while, check is required here.
let's add clock_gettime() here instead (like in code block above) to update ts